[2022-06-16 11:54:15,694] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Job-pts.pts scheduled__2022-06-16T09:11:47.345967+00:00 [queued]>
[2022-06-16 11:54:16,006] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Job-pts.pts scheduled__2022-06-16T09:11:47.345967+00:00 [queued]>
[2022-06-16 11:54:16,006] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 11:54:16,006] {taskinstance.py:1357} INFO - Starting attempt 2 of 6
[2022-06-16 11:54:16,006] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 11:54:16,285] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): pts> on 2022-06-16 09:11:47.345967+00:00
[2022-06-16 11:54:16,553] {standard_task_runner.py:52} INFO - Started process 9807 to run task
[2022-06-16 11:54:17,037] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'Job-pts', 'pts', 'scheduled__2022-06-16T09:11:47.345967+00:00', '--job-id', '176', '--raw', '--subdir', 'DAGS_FOLDER/docker_job/price_scale.py', '--cfg-path', '/tmp/tmp5p680vw7', '--error-file', '/tmp/tmpsmxl2qhv']
[2022-06-16 11:54:17,038] {standard_task_runner.py:80} INFO - Job 176: Subtask pts
[2022-06-16 11:54:21,413] {warnings.py:110} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/configuration.py:525: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2022-06-16 11:55:02,106] {task_command.py:370} INFO - Running <TaskInstance: Job-pts.pts scheduled__2022-06-16T09:11:47.345967+00:00 [running]> on host 2d9e52ec85d2
[2022-06-16 11:55:22,114] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=samuel.momoh-olle@asgoodasnew.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=Job-pts
AIRFLOW_CTX_TASK_ID=pts
AIRFLOW_CTX_EXECUTION_DATE=2022-06-16T09:11:47.345967+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-16T09:11:47.345967+00:00
[2022-06-16 11:55:23,279] {logging_mixin.py:115} INFO - /opt/airflow
[2022-06-16 12:00:00,338] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/docker_job/price_scale.py", line 175, in run_from_class
    python_task.execute(dict())
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/docker_job/price_scale.py", line 115, in test
    ppd = ProductPriceData(driver=webdriver.Chrome(executable_path="dags/docker_job/chromedriver", options=options))
  File "/home/airflow/.local/lib/python3.7/site-packages/selenium/webdriver/chrome/webdriver.py", line 73, in __init__
    service_log_path, service, keep_alive)
  File "/home/airflow/.local/lib/python3.7/site-packages/selenium/webdriver/chromium/webdriver.py", line 98, in __init__
    options=options)
  File "/home/airflow/.local/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 275, in __init__
    self.start_session(capabilities, browser_profile)
  File "/home/airflow/.local/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 365, in start_session
    response = self.execute(Command.NEW_SESSION, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 430, in execute
    self.error_handler.check_response(response)
  File "/home/airflow/.local/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 247, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: DevToolsActivePort file doesn't exist
Stacktrace:
#0 0x56267bf35f33 <unknown>
#1 0x56267bc80118 <unknown>
#2 0x56267bca6282 <unknown>
#3 0x56267bca27c6 <unknown>
#4 0x56267bc9ed5a <unknown>
#5 0x56267bcd9d3a <unknown>
#6 0x56267bcd3e63 <unknown>
#7 0x56267bca982a <unknown>
#8 0x56267bcaa985 <unknown>
#9 0x56267bf7a4cd <unknown>
#10 0x56267bf7e5ec <unknown>
#11 0x56267bf6471e <unknown>
#12 0x56267bf7f238 <unknown>
#13 0x56267bf59870 <unknown>
#14 0x56267bf9b608 <unknown>
#15 0x56267bf9b788 <unknown>
#16 0x56267bfb5f1d <unknown>
#17 0x7fb85509cea7 <unknown>

[2022-06-16 12:00:02,767] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=Job-pts, task_id=pts, execution_date=20220616T091147, start_date=20220616T115415, end_date=20220616T120002
[2022-06-16 12:00:06,284] {standard_task_runner.py:97} ERROR - Failed to execute job 176 for task pts (Message: unknown error: DevToolsActivePort file doesn't exist
Stacktrace:
#0 0x56267bf35f33 <unknown>
#1 0x56267bc80118 <unknown>
#2 0x56267bca6282 <unknown>
#3 0x56267bca27c6 <unknown>
#4 0x56267bc9ed5a <unknown>
#5 0x56267bcd9d3a <unknown>
#6 0x56267bcd3e63 <unknown>
#7 0x56267bca982a <unknown>
#8 0x56267bcaa985 <unknown>
#9 0x56267bf7a4cd <unknown>
#10 0x56267bf7e5ec <unknown>
#11 0x56267bf6471e <unknown>
#12 0x56267bf7f238 <unknown>
#13 0x56267bf59870 <unknown>
#14 0x56267bf9b608 <unknown>
#15 0x56267bf9b788 <unknown>
#16 0x56267bfb5f1d <unknown>
#17 0x7fb85509cea7 <unknown>
; 9807)
[2022-06-16 12:00:11,452] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-06-16 12:00:14,677] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
